### DISCUSSION TITLE
I have an Issue in Vercel.json (I think), and I need a help :(

### DISCUSSION THREAD

**Discussion by GoldManeitor** [vercel/community#5711] [Help] [answered]
### Summary

Hi! I'm trying to deploy a Serverless Function but in the build stage i get this issue:

'Error: The pattern "api/sendMail" defined in `functions` doesn't match any Serverless Functions inside the `api` directory.'

The only purpouse of my function is send emails automatically. So, i have a typescript file called 'SendMail.ts' in a folder (called 'api').
My schema is like this:

/my-project
├── /api
 |      └── sendMail.ts
├── /node_modules
├── .env
├── .vercelignore
├── package.json
└── tsconfig.json

and my vercel.json: 

{
  "functions": {
    "api/sendMail": {
      "memory": 512
    }
  }
}

What is the problem? (sorry for my so bad english and thanks for your time)

### Steps to Reproduce

- create a project
- install dependecies (vercel, node, typescript, nodemailer, resend etc)
- create an api folder
- create a SendMail.ts file
- create a similar function like that: 

        import { VercelRequest, VercelResponse } from '@vercel/node';
        import nodemailer from 'nodemailer';
        import fetch from 'node-fetch';
        
        // Nodemailer and Resend API Key configuration
        const transporter = nodemailer.createTransport({
          service: 'resend', // 
          auth: {
            user: 'myemail@resend', //
            pass: 'mypass', // 
          },
        });
        
        export default async (req: VercelRequest, res: VercelResponse) => {
          try {
            const { email, subject, message } = req.body;
        
            if (!email || !subject || !message) {
              return res.status(400).json({ error: 'Missing data.' });
            }
        
            // send email using Nodemailer
            const mailOptions = {
              from: 'clientemail@gmail.com'
              to: email,
              subject,
              text: message,
            };
        
            await transporter.sendMail(mailOptions);
        
            await fetch('URL_DE_API_RESEND', {
              method: 'POST',
              headers: {
                'Content-Type': 'application/json',
                'Authorization': `Bearer ${process.env.RESEND_API_KEY}`,
              },
              body: JSON.stringify({
                // Resend api data permissions
                // ...
              }),
            });
        
            res.status(200).json({ success: true });
          } catch (error) {
            console.error('Sending email error:', error);
            res.status(500).json({ error: 'Server internal error.' });
          }
        };

- add the above vercel.json code to vercel.json file, after created it
- create a vercelignore file for enviroment variables
- create a project in Vercel page
- add the project, configure enviroment variables and build

**Comment by swarnava** [comment, marked as answer]
Can you try:

```json
{
    "functions": {
        "api/*.*": {
            "memory": 512
        }
    }
}
```

**Reply by GoldManeitor** [reply]
OMG, that was fast and my app is ready hahah you`re a genius. So many thanks!!

**Reply by swarnava** [reply]
No problem. You’re very welcome! :)
